map_view/map-rotation/src/main/java/com/esri/samples/map_rotation/MapRotationSample.java

```diff
@@ -67,41 +66,26 @@ public void start(Stage stage) {
       String yourAPIKey = System.getProperty("apiKey");
       ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
 
-      // create a slider with a range of 360 units and set it to 5 degrees
-      slider = new Slider(-180.0, 180.0, 5.0);
-      styleSlider();
-
-      // listen for the value in the slider to change
-      slider.valueProperty().addListener(e -> {
-        // rotate map view based on new value in slider
-        mapView.setViewpointRotationAsync(slider.getValue());
-      });
-
       // create a map with the streets basemap style
       map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS);
 
-      // enable slider when map view is done loading
-      map.addDoneLoadingListener(() -> {
```

Comment:
since we've removed the done loading listener on the map, it's no longer in danger of being garbage collected before the code inside the logic completes. So we can remove its member variable assignment on line 46 and instantiate it instead on line 70 

```java
      // create a map with the streets basemap style
      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS);
```